### DISCUSSION TITLE
Accidentally pushed assignment-product-cards to class and not personal github

### DISCUSSION THREAD

**Discussion by jackiebgregory** [nss-evening-cohort-14/evening-client-side#39] [Help Me!] [answered]
**Your Name:** Jackie

**Have you searched the other help tickets?** Yes

**Describe your issue.** I have jackiebgregory/ASSIGNMENT-product-cards, and on study group night created the setup branch, pushed the files up, and it was fine. I have been working on the requirements and additional style requirements tickets and realized that I was making all my changes to main and had forgot to create a requirements branch. I added, committed, and pushed it up, but it looks like it didn't go. Then I created a new branch called requirements, added, committed, and pushed it up, and it looks like it went to nss-nightclass-projects/ASSIGNMENT-product-cards instead of jackiebgregory/ASSIGNMENT-product-cards. How can I move it to the right place?

**List any error messages that appear.** 
![Screen Shot 2021-01-18 at 8 29 50 PM](https://user-images.githubusercontent.com/77009672/104980339-edee9080-59cb-11eb-9c50-f25bf2447b7e.png)

**What have you googled?** (Paste links to the things you have found from your google search - You must include at least 3 links)
https://stackoverflow.com/questions/31977100/git-push-is-pushing-to-the-wrong-repo
https://github.community/t/project-attached-to-the-wrong-github-repo/360
https://gist.github.com/carltondickson/28a90ccfed2f6b697aba

**What have you tried? Tell us everything you have tried.**
I tried to push it up again, but I get this message:
assignment-product-cards git:(requirements) git push origin requirements
Everything up-to-date
➜  assignment-product-cards git:(requirements) 

**Copy/paste link TO THE FILE on your branch or in your repo that you are having issues with.**
https://github.com/jackiebgregory/ASSIGNMENT-product-cards

**Comment by jackiebgregory** [comment]
Now I'm wondering if maybe the updates were only saved in Visual Studio code and not my local terminal? I think when I returned to working on it, I went right to VSCode and not the terminal....

**Comment by TrinityChristiana** [comment]
Can you send the response of this command in your local Product Cards repo? 

```terminal
git remote -v
```

**Reply by jackiebgregory** [reply]
➜  ASSIGNMENT-product-cards git:(main) git remote -v
origin  git@github.com:jackiebgregory/ASSIGNMENT-product-cards.git (fetch)
origin  git@github.com:jackiebgregory/ASSIGNMENT-product-cards.git (push)
➜  ASSIGNMENT-product-cards git:(main)

**Comment by TrinityChristiana** [comment]
So it looks like the changes you made were pushed to the main branch of your repo. Here's the commit: https://github.com/jackiebgregory/ASSIGNMENT-product-cards/commit/003bb446138b44da46cd474284bdaa306f25b3a3. Are your changes there?

**Reply by jackiebgregory** [reply]
Yes, my changes are there. How do I get the link to put in the Requirements issue ticket so I can close that one out? I just tried it, and now I see this: 
![Screen Shot 2021-01-20 at 8 29 50 PM](https://user-images.githubusercontent.com/77009672/105272245-8c126000-5b5e-11eb-8914-3685a945d5a4.png)

Why does it look like its forked from Holly's when I didn't do that? I tried to change it to compare my requirements to my main and it said there was nothing to compare. But it seems that my changes did get pushed up?

**Reply by drteresavasquez** [reply]
Don't worry about the fork. Only focus on your repo. Follow the same path we do in class. Make a branch commit the branch. Push the branch. Create a PR for the branch against YOUR main branch, merge the branch. In your local, checkout main. Pull main. Create a new branch and start the process over again. You should be good. To close an issue ticket after you're done, select close issue ticket. 

To get the link to your repo, copy it from your browser address bar. 

Also, let's keep these discussion tickets focused on one topic. If you find you have a different question, create a new discussion topic.

**Comment by drteresavasquez** [comment]
I marked this as answered because you are connected to your repo and your should know how to take it from here

**Reply by jackiebgregory** [reply]
I am still lost on this one, but working on it!

**Comment by drteresavasquez** [comment]
@jackiebgregory did you resolve this? If you follow the steps @TrinityChristiana gave to check the remote and it's pointing to your repo, make a branch, make a change, add and commit and push it up to GitHub and see what happens. The change can be something small. 

Let us know if it doesn't work.

**Comment by jackiebgregory** [comment]
No, I'm still stuck on this. I made a few small changes, added, committed, and pushed it up. It doesn't seem like it went through - there are no pull requests.

![Screen Shot 2021-01-22 at 5 06 05 PM](https://user-images.githubusercontent.com/77009672/105558502-568f8300-5cd4-11eb-9bc5-03c2fd0b64fc.png)

**Reply by jackiebgregory** [reply]
Tried again, creating a new branch called requirements2. I think I might have accidentally pushed it to someone else's github. 
![Screen Shot 2021-01-22 at 5 15 18 PM](https://user-images.githubusercontent.com/77009672/105558963-91de8180-5cd5-11eb-98ae-227c671f16be.png)

**Comment by TrinityChristiana** [comment, marked as answer]
It's being pushed up to the correct repo. Here's your requirements2 branch on your repo: https://github.com/jackiebgregory/ASSIGNMENT-product-cards/tree/requirements2

While creating a pull request make sure to change the base directory to your own repo. When you select your repo it will change the compare branch to main. Select the correct branch you want merged into main as the compare branch.

![image](https://user-images.githubusercontent.com/31781724/105561484-73c84f80-5cdc-11eb-8d4f-f0c17a04706c.jpeg)

Change the circled drop down to your own repo.

**Reply by TrinityChristiana** [reply]
By default it might have the originally forked repo as the base repo but I think after changing it it shouldn't change back.

**Reply by TaylorHardaway** [reply]
I'm reading this ticket, because this has happened to me twice now. I remember now that there is a spot where you can change it back to your own repo but after I push it it does seem to go back to the group repo first, and then I have to change it.

**Reply by drteresavasquez** [reply]
We can delete it. Just create a new PR against your branch in the new PR.
